app/src/main.c

```diff
@@ -744,9 +866,9 @@ int main(void)
 		return 0;
 	}
 
-	err = bt_bap_broadcast_source_get_id(broadcast_source, &broadcast_id);
-	if (err != 0) {
-		printk("Unable to get broadcast ID: %d\n", err);
+	err = bt_rand(&broadcast_id, BT_AUDIO_BROADCAST_ID_SIZE);
```

Comment:
this logic change probably also relates to zephyr version used newer than the one linked in the west.yml in the project... (maybe we should just bump that.. and test all configs)

Reply:
Saw some issues with the new zephyr version, so lets stick with the old version for now.